src/signals/settings/roles/containers/RolesOverview/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { bindActionCreators } from 'redux';
+import { Row, Column } from '@datapunt/asc-ui';
+
+import makeSelectRolesModel from 'models/roles/selectors';
+import { fetchRoles } from 'models/roles/actions';
+
+import RolesList from '../../components/RolesList';
+
+export const RolesOverview = ({
+  roles: {
+    list,
+    loading,
+  },
+  onFetchRoles,
+}) => {
+  useEffect(() => {
+    onFetchRoles();
+  }, []);
+
+  return (
+    <div>
+      <Row>
+        <Column span={12}>
+          <RolesList
+            list={list}
+            loading={loading}
```

Comment:
Shouldn't the loading state be handled by this container component instead of individual, underlying, components?

Reply:
dat heb ik helemaal eruit gehaald in de ander branch /SIG-1697-als-beheerder-wil-ik-een-rol-kunnen-bewerken